course_discovery/apps/course_metadata/data_loaders/api.py

```diff
@@ -1016,6 +1016,15 @@ def _add_course_subjects(self, categories, course_run):
                 subject.slug = category['slug']
                 subject.save()
 
+            if subject:
+                for language_code, translated_title in category['title_translations'].items():
```

Comment:
will `title_translations` key always be present in `category` dict?  we could use `get` method.

Reply:
Done